### PR TITLE
Sync `master` with `develop`

### PR DESCRIPTION
**Pull request by Gudahtt**
This brings `develop` up to date with the `v8.1.4` release.

This PR also includes a revert of the revert of the new main quote designs (i.e. they have been restored).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,40 @@
 ## Current Develop Branch
 - [#9612](https://github.com/MetaMask/metamask-extension/pull/9612): Update main-quote-summary designs/styles
 
+## 8.1.4 Tue Nov 10 2020
```
Comment (rekmarks) [inline CHANGELOG.md:6]: This ain't right

**Review by rekmarks — APPROVED** [review]
The date on the changelog is wrong, but I don't know that we care

File: CHANGELOG.md (hunk 1 above)
Comment (Gudahtt) [reply CHANGELOG.md:6]: All of the dates in this changelog are wrong :sweat_smile:  The date is generated when the changelog is updated.

I think we should update the changelog script to omit a date altogether. Then maybe add it manually in this sync PR. I'll leave that for another day - the real dates can be applied retroactively by looking at the GitHub releases at least.

Comment (Gudahtt) [reply CHANGELOG.md:6]: Oh and once we automate our release process, this step can be automated as well.

**Pull request merged**